### ISSUE TITLE
Extra/empty figures MultiSim.plot()

### ISSUE THREAD

**Issue by krosenfeld-IDM** [InstituteforDiseaseModeling/covasim#244]
**Describe the bug**

There appear to be some extra axes when plotting a MultiSim

**To reproduce**

```
import covasim as cv
msim = cv.MultiSim(cv.Sim())
msim.run(verbose=False)
msim.plot();
```

_For model issues:_
```python
import covasim as cv
sim = cv.Sim(pars={}) # Configure the simulation
sim.run() # Steps required to produce the behavior
``

**Screenshots or outputs**


![image](https://user-images.githubusercontent.com/56368533/82961388-14117d00-9f72-11ea-8295-d1180d29b968.png)


**Platform (please complete the following information):**
 - Covasim version: 1.3.2
 - OS: Linux
 - Other platform information: jupyter notebook

**Comment by ckerr-IDM** [comment]
@krosenfeld-IDM i've seen this too, it's an issue with jupyter notebooks...i'm not sure how to fix it, but if you run the same commands from a script you get this:
![image](https://user-images.githubusercontent.com/53099813/82964419-47f0a080-9f7a-11ea-934c-e0a81578ebb4.png)

**Comment by cliffckerr** [comment]
@krosenfeld-IDM ok, i figured out the issue. from a script, this code produces a single plot:
```
import matplotlib.pyplot as plt
fig = plt.figure()
ax  = fig.add_subplot()
ax.plot([1,3,4])
plt.show()
plt.plot([5,6,6])
plt.show()
```

![image](https://user-images.githubusercontent.com/3239256/82966149-fac2fd80-9f7e-11ea-9a82-14b7339c9356.png)

but in a notebook, it produces two separate plots:
![image](https://user-images.githubusercontent.com/3239256/82966185-17f7cc00-9f7f-11ea-9028-20b2fb794e4a.png)

so i think the issue is to ensure that `do_show=False` is passed except on the last iteration. i'll do that now

**Comment by cliffckerr** [comment]
Closed by #567(AMath)
